airbyte-webapp/src/views/Connector/ServiceForm/utils.ts

```diff
@@ -3,8 +3,8 @@ import { naturalComparator } from "utils/objects";
 
 import { ConnectorDefinitionSpecification } from "../../../core/domain/connector";
 
-export function makeConnectionConfigurationPath(path: string[]): string {
-  return `connectionConfiguration.${path.join(".")}`;
+export function makeConnectionConfigurationPath(path: string[] = []): string {
+  return ["connectionConfiguration", ...path].join(".");
```

Comment:
:information_source: Making sure this method also works with an empty `path` array passed in.